annotate.el

```diff
@@ -665,30 +703,40 @@ to 'maximum-width'."
   (save-excursion
     (goto-char (1- (point))) ; we start at the start of the next line
     ;; find overlays in the preceding line
-    (let ((prefix (annotate-make-prefix)) ; white space before first annotation
-          (bol (progn (beginning-of-line) (point)))
-          (eol (progn (end-of-line) (point)))
-          (text "")
-          (overlays nil))
+    (let ((prefix          (annotate-make-prefix)) ; white space before first annotation
+          (bol             (progn (beginning-of-line) (point)))
+          (eol             (progn (end-of-line) (point)))
+          (text            "")
+          (overlays        nil)
+          (face-type-count 1))
```

Comment:
why is it a face-*type*-count? I think I would simply call this `annotation-counter` or even just `idx`.

Reply:
``idx`` look too general to me, ``annotation-counter`` looks fine, though! :)